browser/skus/skus_service_factory.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_SKUS_SKUS_SERVICE_FACTORY_H_
+#define BRAVE_BROWSER_SKUS_SKUS_SERVICE_FACTORY_H_
+
+#include "base/memory/singleton.h"
+#include "brave/components/skus/common/skus_sdk.mojom.h"
+#include "components/keyed_service/content/browser_context_keyed_service_factory.h"
+#include "mojo/public/cpp/bindings/pending_remote.h"
+
+class BraveVpnServiceFactory;
```

Comment:
this isn't needed anymore

Reply:
Removed in https://github.com/brave/brave-core/pull/10847/commits/d634f38c925efbd98b7fe4cb274ae77d102b431d